spacy/displacy/templates.py

```diff
@@ -51,7 +51,9 @@
 TPL_ENT = """
 <mark class="entity" style="background: {bg}; padding: 0.45em 0.6em; margin: 0 0.25em; line-height: 1; border-radius: 0.35em;">
     {text}
-    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">{label}</span>
+    <span style="font-size: 0.8em; font-weight: bold; line-height: 1; border-radius: 0.35em; vertical-align: middle; margin-left: 0.5rem">{label}
+    <a style="text-decoration: none; color: black; font-weight: bold" href="{kb_url}">{kb_id}</a>
```

Comment:
The problem I see with this is that the markup here will be added to _all_ visualizations, even those that don't have a `kb_id` set (which is very common because not all users train an entity linker). So if we want to include this, we probably want to construct the link outside of the template and pass it in if a `kb_id` is available – otherwise, the variable receives an empty string.

Reply:
I did test this with an empty string and it'd basically wouldn't be displayed, it would just be an empty `<a>` element, so I'm not sure it's worth the hassle? But yea it might be cleaner to have this in a conditional.

Reply:
We shouldn't be generating empty links as part of the default markup, though – it adds a lot of bloat for no reason, is potentially confusing and also produces unsemantic HTML.